api_by_features/battery.json

```diff
@@ -0,0 +1,45 @@
+{
+    "Baterry": {
+    	"description": "Ability to read level and mode of charge of baterry.",
+        "browser": {
+        	"API": "Baterry Status API",
+        	"spec_url": "http://www.w3.org/TR/battery-status/",
+        	"working group": ""
+        },
+        "sysapp": "browser",
```

Comment:
I recommend getting rid of "sysapp" entries since they're unlikely to be relevant now.